Makefile

```diff
@@ -176,27 +176,31 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 .PHONY: deploy
 deploy: manifests kustomize ## Deploy manager to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	kubectl --context ${DEFAULT_IMBS_CONTEXT} apply -f config/crd/bases/control.kubestellar.io_workstatuses.yaml
 	$(KUSTOMIZE) build config/default | kubectl --context ${DEFAULT_IMBS_CONTEXT} apply -f -
 
 .PHONY: undeploy
 undeploy: ## Undeploy manager from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/default | kubectl --context ${DEFAULT_IMBS_CONTEXT} delete --ignore-not-found=$(ignore-not-found) -f -
+	kubectl --context ${DEFAULT_IMBS_CONTEXT} delete --ignore-not-found=$(ignore-not-found) -f config/crd/bases/control.kubestellar.io_workstatuses.yaml
 
 .PHONY: chart
 chart: manifests kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(shell echo ${IMG} | sed 's/\(:.*\)v/\1/')
 	$(KUSTOMIZE) build config/default > chart/templates/operator.yaml
+	@mkdir -p chart/crds
+	$(KUSTOMIZE) build config/crd > chart/crds/crds.yaml
```

Comment:
This is a problem, see https://github.com/kubestellar/ocm-status-addon/blob/7521a376c0eb8961e12668f4c5f9f643ee3a8769/config/crd/kustomization.yaml#L1-L2

Reply:
Done.